.github/workflows/benchmarks.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - main
   pull_request:
+  workflow_dispatch:
```

Comment:
I believe `workflow_dispatch` is only allowed for write accessed users (i.e. safe for open-source repo) [[link](https://github.com/orgs/community/discussions/26622)].